app/router.ts

```diff
@@ -3,6 +3,12 @@ import { inject as service } from '@ember/service';
 import config from 'ember-get-config';
 import { Blocker } from 'ember-osf-web/services/ready';
 
+const {
+    featureFlags: {
+        routes,
+    },
+} = config as { featureFlags: { routes: { [index: string]: string } } }; // eslint-disable-line no-use-before-define
```

Comment:
The `eslint-disable-line no-use-before-define` is due to https://github.com/eslint/typescript-eslint-parser/issues/435